newIDE/app/src/SceneEditor/index.js

```diff
@@ -459,6 +460,17 @@ export default class SceneEditor extends React.Component<Props, State> {
     }
   };
 
+  _onCreateNewObjectAndFirstInstanceUnderCursor = () => {
```

Comment:
"First" seems unnecessary. Also, avoid "on" in the name of methods that are actually doing actions. Prefer the verb:

`_createNewObjectAndInstanceUnderCursor`